pkg/resources/marketplace/manifestHelper.go

```diff
@@ -49,6 +74,113 @@ func GenerateRegistryConfigMapFromManifest(manifestPackageName string) (map[stri
 	return configMapData, nil
 }
 
+func GetCRDsFromManifestAsStringList(dir string, regex string, packageYaml string, manifestPackageName string, manifestDir string) (string, error) {
+	var stringList strings.Builder
+	libRegEx, e := regexp.Compile(regex)
+	if packageYaml != "" {
+		if e != nil {
+			logrus.Fatalf("Error compiling regex for registry file: %s", e)
+		}
+
+		var folders []string
+
+		e = filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
+			if info.IsDir() && info.Name() != manifestPackageName {
+				folders = append(folders, path)
+			}
+			return nil
+		})
+
+		ReverseSlice(folders)
```

Comment:
If I understand what is going on here correctly. The result of the ReverseSlice would give us an order list going from large to small in name ie `[10.0.0, 9.4.1, 9.4.0]`. There maybe an issue with this. The rhsso is moving from the up stream version to down stream version. That would mean the folder name for the latest version is `7.4.1` which is replacing `9.0.3`. In the crds there are different operators referenced and that's how the version numbers can go back awards.

Reply:
This is a valid concern, but I think it's such an unlikely event, that we would catch in Delorean in any case that I'd prefer to leave this as a known issue, and correct it if it occurs, rather than spending the time addressing it now.

Reply:
Yap I agree with that. Its much better to have the known issue, than the unknown issue.

Reply:
You point brought up a good point on the version numbers so I've updated to pull the version number from the package yaml file rather than the highest number of directory. We'll still have the known issue but we'll be adding the correct CRDs for the current version correctly now.